kornia/utils/grid.py

```diff
@@ -3,6 +3,7 @@
 import torch
 
 
+@torch.jit.ignore
```

Comment:
why is really needed ?

Reply:
`create_meshgrid` doesn't seem to be JITable in its current form, which prevents jit.script working for `spatial_expectation_2d` and `spatial_soft_argmax2d`.

Reply:
ok, then we need to investigate meshgrid with jit, If you could open as issue fort that would be great.

Reply:
I figured out a fix, it was really simple (just replace the questionable usage of `Optional[bool]` with `bool`).